common/utils/ti_fec.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "board_config_common.h"
+
+#define DEBUG_PRINTF(a...) \
```

Comment:
Probably should be removed, if just for debugging.

Reply:
It's also for profiling? IDK, i think it's worth keeping as a empty macro